src/repository/comments.py

```diff
@@ -37,3 +56,66 @@ async def create_comment (
     db.refresh(comment)
     return comment
     
+
+async def get_comments(
+        photo_id: int, 
+        limit: int, 
+        offset: int,
+        db: Session,
+        ) -> List[PhotoComment]:
+    """
+    The get_comments function returns a list of comments for the photo with the given id.
+        The limit and offset parameters are used to paginate through results.
+        
+    :param photo_id: int: Filter the comments by photo id
+    :type photo_id: int
+    :param limit: int: Limit the number of comments returned
+    :type limit: int
+    :param offset: int: Specify the number of records to skip before starting to return rows
+    :type offset: int
+    :param db: Session: Pass the database session to the function
+    :type db: Session
+    :return: A list of photocomment objects
+    :rtype: List[PhotoComment] 
+    """
+    stmt = Select(PhotoComment).filter_by(photo_id=photo_id).order_by(asc(PhotoComment.id)).offset(offset).limit(limit)
+    contacts =  db.execute(stmt)
+    return contacts.scalars().all()
```

Comment:
Expected type 'list[PhotoComment]', got 'Sequence[Row | RowMapping | Any]' instead

Reply:
changed to 'Sequence'